examples/flax_mnist.py

```diff
@@ -15,16 +15,32 @@
 """MNIST example with Flax and JAXopt."""
 
 from absl import app
+from absl import flags
+
 from flax import linen as nn
+
 import jax
 import jax.numpy as jnp
+
 from jaxopt import loss
 from jaxopt import OptaxSolver
+from jaxopt import PolyakSGD
 from jaxopt import tree_util
+
 import optax
+
 import tensorflow_datasets as tfds
 
 
+flags.DEFINE_float("l2_regul", 1e-4, "L2 regularization.")
+flags.DEFINE_float("learning_rate", 0.001, "Learning rate (ADAM).")
+flags.DEFINE_bool("manual_loop", False, "Whether to use a manual loop.")
```

Comment:
description could be more informative. A manual loop for what? Also, I don't understand (yet) what manual refers to